package.json

```diff
@@ -32,32 +32,32 @@
   "dependencies": {
     "cliui": "^3.2.0",
     "eslint-rule-documentation": "^1.0.0",
-    "path-is-absolute": "1.0.0",
+    "path-is-absolute": "1.0.1",
```

Comment:
I don't know. This might have been introduced because of one's personal preference using `--save-exact` or sth.  
Eventually, this should be uniform. I suggest we change all the dependencies (see devDeps) to also use the `^` soon.